src/Illuminate/Support/helpers.php

```diff
@@ -195,7 +195,7 @@ function array_has($array, $key)
      * @param  mixed  $default
      * @return mixed
      */
-    function array_last($array, $callback, $default = null)
+    function array_last($array, $callback = null, $default = null)
```

Comment:
I noticed this too but opted to leave it since it wasn't directly related to the proposed change. Can add it to this PR if requested though.